release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1819,6 +1825,11 @@ In the table below, features are marked with the following statuses:
 |-
 |TP
 
+|HTTP/2 Support
+|-
+|-
+|GA
+
```

Comment:
I don't think we need this entry in the table if the feature was never labeled "TP".